software-engineering/transparency-and-security.md

```diff
@@ -61,15 +62,73 @@ Example: https://github.com/Open-Systems-Pharmacology/Suite/commits/master
   * Test protocols of all automated tests
   * All produced build artifacts
   
-Example: https://github.com/Open-Systems-Pharmacology/Suite/actions
+
+Example: [https://github.com/Open-Systems-Pharmacology/Suite/actions/workflows/build-and-publish.yml](https://github.com/Open-Systems-Pharmacology/Suite/actions/workflows/build-and-publish.yml)
 
 * Build history entries cannot be modified
 
 ### Software validation and qualification
-An overview of validation steps and links to validation/test reports are published with every OSP Suite release on GitHub (https://github.com/Open-Systems-Pharmacology/Suite/tree/master/validation%20and%20qualification) E.g., the test reports of the OSP Suite version 8 contain more than 10,000 tests.
+An overview of validation steps and links to validation/test reports are published with every OSP Suite release on GitHub ([https://github.com/Open-Systems-Pharmacology/Suite/tree/master/validation%20and%20qualification](https://github.com/Open-Systems-Pharmacology/Suite/tree/master/validation%20and%20qualification)) E.g., the test reports of the OSP Suite version 8 contain more than 10,000 tests.
 
 Qualification reports of the OSP platform are published on GitHub. Examples:
 
-* https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_GFR_Ontogeny/releases
-* https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_CYP3A4_Ontogeny/releases
-* https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_CYP2C8_Ontogeny/releases
\ No newline at end of file
+* [https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_GFR_Ontogeny/releases](https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_GFR_Ontogeny/releases)
+* [https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_CYP3A4_Ontogeny/releases](https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_CYP3A4_Ontogeny/releases)
+* [https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_CYP2C8_Ontogeny/releases](https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_CYP2C8_Ontogeny/releases)
+
+## Security 
```

Comment:
The main change - new section **Security**
Preview: https://github.com/Yuri05/developer-docs/blob/add_security_section_fix_links/software-engineering/transparency-and-security.md#security